tooling/nargo_fmt/src/visitor/expr.rs

```diff
@@ -39,6 +39,16 @@ impl FmtVisitor<'_> {
                     self.format_expr(infix.rhs)
                 )
             }
+            ExpressionKind::Call(call_expr) => {
+                let formatted_func = self.format_expr(*call_expr.func);
+                let formatted_args = call_expr
+                    .arguments
+                    .iter()
```

Comment:
Thank you, cargo fmt. `.arguments` is completely worthy of being on its own separate line.